petergreve/application/app/key.js

```diff
@@ -0,0 +1,3 @@
+const key = 'AIzaSyB6LYUvbFC73pLDjn-FhpcEvSgJ48f3Ag4'
+
+export default key;
```

Comment:
You should have not included the api secret key in an open-source repo for everyone to see.